roles/contiv_network/files/v2plugin.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -x
+sleep 2
```

Comment:
What is the purpose of the sleep here? If it is to let the plugin to start, can we make it loop on that condition?

Reply:
To make sure netplugin would init completely before fire this script. During unit testing saw , this command getting no op if script exec before netplugin complete its init

Reply:
Is there some state we can poll instead of sleeping?